libraries/common/src/lib/feature/content/feature-content.component.html

```diff
@@ -45,8 +45,13 @@ <h1>Pipe</h1>
 		</div>
 		<div class="grid-xxl4 u-paddingLeftBig">
 			<nav class="feature-quickNav">
-				<a class="feature-quickNav-item" *ngIf="infos.examples && infos.examples.length" routerLink="./"
-					fragment="examples" queryParamsHandling="preseve">Design</a>
+				<ng-container *ngIf="infos.examples && infos.examples.length">
+					<a class="feature-quickNav-item" routerLink="./"
+						fragment="examples" queryParamsHandling="true">Design</a>
+						<a href="" class="feature-quickNav-item mod-subItem" *ngFor="let example of infos.examples" routerLink="./"
```

Comment:
Put the routerlink parameter in the a href ?

Reply:
Routerlink does it 
![image](https://user-images.githubusercontent.com/26228000/93453190-a6e77600-f8d9-11ea-9b91-d17c41ade60f.png)